environment/directories.rst

```diff
@@ -50,6 +47,39 @@ You can use :ref:`filesystem-project` for larger datasets.
    If you also have an account on `lsst-dev <https://developer.lsst.io/services/lsst-dev.html>`_,
    you can access your Notebook Aspect home directory via :file:`$HOME/jhome/` from lsst-dev.
 
+Preinstalled subdirectories
+---------------------------
+
+Within the :file:`$HOME` directory a number of directories are currently pre-created for new users.
+
+:file:`~/notebooks`
```

Comment:
Make this a title

```suggestion
The ~/notebooks directory
^^^^^^^^^^^^^^^^^^^^^^^^^
```

(and no formatting in a title)

Reply:
I wasn't sure we actively used titles that far down, but with that encouragement: I actually prefer it.